modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java

```diff
@@ -91,6 +91,16 @@ public void setInlineSchemaNameDefaults(Map inlineSchemaNameDefaults) {
         } else {
             // not set so default to null;
         }
+
+        if (this.inlineSchemaNameDefaults.containsKey("RESOLVE_INLINE_ENUMS")) {
+            if (Boolean.valueOf(this.inlineSchemaNameDefaults.get("RESOLVE_INLINE_ENUMS"))) {
+                this.resolveInlineEnums = true;
```

Comment:
You can remove the if statement
`this.resolveInlineEnums = Boolean.valueOf(this.inlineSchemaNameDefaults.get("RESOLVE_INLINE_ENUMS"))`

Reply:
fixed